src/progressive-profiling/ProgressiveProfiling.jsx

```diff
@@ -45,7 +47,7 @@ const ProgressiveProfiling = (props) => {
   const [values, setValues] = useState({});
   const [openDialog, setOpenDialog] = useState(false);
   const [showRecommendationsPage, setShowRecommendationsPage] = useState(false);
-
+  const authenticatedUser = getAuthenticatedUser();
```

Comment:
Will this cause any error if the user doesn't exit i.e some unauthenticated user tries to open the `/welcome` page

Reply:
nope. it will just return `null` to authenticated user and the `ensure_authenticated_user` function will redirect the user to login page